_Group Projects/Group_5/README.md

```diff
@@ -0,0 +1,21 @@
+[R] group5 - user program
+
+SYNTRAX
+
+group5 [depvar] [indepvar]
+
+OVERVIEW
+
+Our folder contains two .do files: program.do and demonstration.do. The former is the code for the user-created program group5. The latter demonstrates example uses of the group5 program. The user must specify the dynamic file path to use the demonstration.do file, per the assignment instructions.
+
+The program group5 loads a default data set called nlswork. This is the National Longitudinal Survey of Young Working Women, which is a time series from 1968 to 1988 of of 4,711 women. The data set contains both numeric and string variables. The group5 program only accepts only numeric variables.
```

Comment:
I like the explanation here a lot. Helps the user more thoroughly understand what the program does and how to use it properly.